kratos/tests/test_import_model_part_modeler.py

```diff
@@ -0,0 +1,193 @@
+import os
+
+# Importing the Kratos Library
+import KratosMultiphysics
+import KratosMultiphysics.KratosUnittest as KratosUnittest
+import KratosMultiphysics.kratos_utilities as KratosUtilities
+from KratosMultiphysics.modelers.import_mdpa_modeler import ImportMDPAModeler
+
+def GetFilePath(filename):
+    return os.path.join(os.path.dirname(os.path.realpath(__file__)), filename)
+
+class TestImportMDPAModeler(KratosUnittest.TestCase):
+    def tearDown(self):
+        # Clean up temporary files
+        KratosUtilities.DeleteFileIfExisting(GetFilePath("test_model_part_io_write.out.mdpa"))
+        KratosUtilities.DeleteFileIfExisting(GetFilePath("test_model_part_io_write.out.time"))
+        KratosUtilities.DeleteFileIfExisting(GetFilePath("test_model_part_io_write.time"))
+        KratosUtilities.DeleteFileIfExisting(GetFilePath("test_model_part_io_write_mesh_only.out.mdpa"))
+        KratosUtilities.DeleteFileIfExisting(GetFilePath("test_model_part_io_write_mesh_only.out.time"))
+        KratosUtilities.DeleteFileIfExisting(GetFilePath("test_model_part_io_write_mesh_only.time"))
+
+    def testImportMDPAModeler(self):
+        # Set up the import model part modeler
+        model = KratosMultiphysics.Model()
+        settings = KratosMultiphysics.Parameters('''{
+            "input_type" : "mdpa",

```

Comment:
This should be part of the name or "use_input_model_part":

```Json
{
    "input_filename" : "",
    "model_part_name" : "Main.mdpa"
}
```

Like that no?

Reply:
Sorry, I forgot updating the test. I think we agreed to use `Main` as `mdpa` is assumed.